plugins/modules/rds_instance.py

```diff
@@ -1115,7 +1116,7 @@ def get_options_with_changing_values(client, module: AnsibleAWSModule, parameter
             )
 
     # Validate changes to storage type options
-    if instance.get("StorageType") == "io1":
+    if instance.get("StorageType") in ["io1", "io2"]:
```

Comment:
```suggestion
    if instance.get("StorageType") in ("io1", "io2"):
```
It is faster to use for this use case, but this is not mandatory since you only have 2 elements here